services/brig/test/integration/Federation/End2end.hs

```diff
@@ -114,22 +112,6 @@ spec _brigOpts mg brig galley cargohold cannon _federator brigTwo galleyTwo carg
 -- | brig |  http2  |federator| http2  |federator|   http   | brig |
 -- |      +-------->+         +------->+         +--------->+      |
 -- +------+         +-+-------+        +---------+          +------+
-testHandleLookup :: Brig -> Brig -> Http ()
-testHandleLookup brig brigTwo = do
-  -- Create a user on the "other side" using an internal brig endpoint from a
-  -- second brig instance in backendTwo (in another namespace in kubernetes)
-  (handle, userBrigTwo) <- createUserWithHandle brigTwo
-  -- Get result from brig two for comparison
-  let domain = qDomain $ userQualifiedId userBrigTwo
-  resultViaBrigTwo <- getUserInfoFromHandle brigTwo domain handle
-
-  -- query the local-namespace brig for a user sitting on the other backend
-  -- (which will exercise the network traffic via two federators to the remote brig)
-  resultViaBrigOne <- getUserInfoFromHandle brig domain handle
-
-  liftIO $ assertEqual "remote handle lookup via federator should work in the happy case" (profileQualifiedId resultViaBrigOne) (userQualifiedId userBrigTwo)
-  liftIO $ assertEqual "querying brig1 or brig2 about the same user should give same result" resultViaBrigTwo resultViaBrigOne
-
 testSearchUsers :: Brig -> Brig -> Http ()
```

Comment:
this isn't called anywhere, but neither is it translated into `/integration`.  i think running this won't pass for the same reason `testHandleLookup` fails?

Reply:
Deleted it now.
The test already existed in the new test suite: https://github.com/wireapp/wire-server/blob/1c6f7463d9eacfe68cdc402952e41213869267c0/integration/test/Test/Brig.hs#L144-L160